src/components/RenderConfig/ColormapConfigComponent/ColormapConfigComponent.tsx

```diff
@@ -46,7 +46,7 @@ export class ColormapConfigComponent extends React.Component<ColormapConfigProps
                     <ScalingSelectComponent selectedItem={renderConfig.scaling} onItemSelect={renderConfig.setScaling} />
                 </FormGroup>
                 <FormGroup label={"Colormap"} inline={true}>
-                    <ColormapComponent inverted={renderConfig.inverted} selectedItem={renderConfig.colorMap} onItemSelect={renderConfig.setColorMap} />
+                    <ColormapComponent inverted={renderConfig.inverted} selectedItem={renderConfig.colorMap} onItemSelect={renderConfig.setColorMap} enableAdditionalColor={true} />
```

Comment:
Is enableAdditionalColor always true? Do we add settings to turn it off in the future?

Reply:
Currently, we don't want CatalogOverlayPlot, VectorOverlay, and ScatterPlot to have newly added colormaps, so I use enableAdditionalColor to control whether the newly added colormaps are allowed. Any suggestion to better write this?